src/main/resources/db/migration/V1_2__prisoner_event_sent.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM prisoner_event_hashes;
+
+ALTER TABLE prisoner_event_hashes ALTER COLUMN prisoner_hash TYPE varchar(24);
```

Comment:
Now stores a Base64 of the MD5 hash instead of the integer generated by Objects.hashCode()